app/views/jobseekers/job_applications/qualifications/edit.html.slim

```diff
@@ -1,14 +1,14 @@
-- content_for :page_title_prefix, t(".title")
+- content_for :page_title_prefix, t(".title.#{secondary? ? 'other' : 'one'}")
 
 .govuk-main-wrapper
   .govuk-grid-row
     .govuk-grid-column-two-thirds
       .govuk-caption-l = t(".caption")
       h1.govuk-heading-xl = t(".heading.#{category}")
 
-      = form_for form, url: jobseekers_job_application_qualification_path(job_application, qualification), method: :patch do |f|
-        = f.govuk_error_summary link_base_errors_to: :subject
+      = form_for form, url: jobseekers_job_application_qualifications_path(job_application, ids: qualifications.pluck(:id)), method: :patch do |f|
+        = f.govuk_error_summary
 
         = render "fields", f: f
 
-        = f.govuk_submit submit_text
+        = f.govuk_submit t("buttons.save_qualification.#{secondary? ? 'other' : 'one'}")
```

Comment:
is this a slight misuse of other/one? im guessing you are saying that you can have many gcse's/ a levels from a school but one degree from a place? i guess most people with degrees do but that doesnt have to be the case?

Reply:
The design forces you to only be able to add degrees (and similar things) one at a time; in cases like that, I'll use the singular here. There isn't really anything we can pass to `count:` - I could pass a `1` or a `2` but that feels quite arbitrary and obscures my intention.

Reply:
ok fair enough, kind of makes sense